src/decanter/core/core_api/train_input.py

```diff
@@ -167,6 +167,13 @@ def __init__(
             input_spec=input_spec
         )
 
+    @staticmethod
```

Comment:
why do you use `staticmethod`?

Reply:
It would be nice for the user to have a list of available TS algorithms before instantiating a TrainTSInput object, or before doing any work in general.